packages/react-composites/src/composites/MeetingComposite/MeetingCallControlBar.tsx

```diff
@@ -19,17 +19,23 @@ export interface MeetingCallControlBarProps {
   peopleButtonChecked: boolean;
   onChatButtonClicked: () => void;
   onPeopleButtonClicked: () => void;
+  mobileView?: boolean;
 }
 
 /**
  * @private
  */
 export const MeetingCallControlBar = (props: MeetingCallControlBarProps): JSX.Element => {
+  const callControlsOptions: CallControlOptions = {
```

Comment:
Shouldn't the `CallControlOptions` be exposed from both the `CallComposite` and `MeetingComposite`?


Perhaps override the default value for `participantsButton` to `false`, but expose the options still?

Reply:
They should but they weren't added to meetings when they were added to CallComposite. I'll make a future work item for this